src/test/java/com/google/firebase/messaging/MessageTest.java

```diff
@@ -417,12 +421,15 @@ public void testApnsMessageWithPayloadAndAps() throws IOException {
         "aps", ImmutableMap.<String, Object>builder()
             .put("alert", ImmutableMap.<String, Object>builder()
                 .put("title", "test-title")
+                .put("subtitle", "test-subtitle")
                 .put("body", "test-body")
                 .put("loc-key", "test-loc-key")
                 .put("action-loc-key", "test-action-loc-key")
                 .put("title-loc-key", "test-title-loc-key")
+                .put("subtitle-loc-key", "test-subtitle-loc-key")
                 .put("loc-args", ImmutableList.of("arg1", "arg2", "arg3"))
                 .put("title-loc-args", ImmutableList.of("arg4", "arg5", "arg6"))
+                .put("subtitle-loc-args", ImmutableList.of("arg7", "arg8", "arg9"))
```

Comment:
Also add a test case for invalid message? Lines 482-485

Reply:
done